ipam/api.go

```diff
@@ -9,21 +9,20 @@ import (
 
 var (
 	// Error responses returned by AddressManager.
-	errInvalidAddressSpace     = fmt.Errorf("Invalid address space")
-	errInvalidPoolId           = fmt.Errorf("Invalid address pool")
-	errInvalidAddress          = fmt.Errorf("Invalid address")
-	errInvalidScope            = fmt.Errorf("Invalid scope")
-	errInvalidConfiguration    = fmt.Errorf("Invalid configuration")
-	errAddressPoolExists       = fmt.Errorf("Address pool already exists")
-	errAddressPoolNotFound     = fmt.Errorf("Address pool not found")
-	errAddressPoolInUse        = fmt.Errorf("Address pool already in use")
-	errAddressPoolNotInUse     = fmt.Errorf("Address pool not in use")
```

Comment:
is these errors not used ?

Reply:
They are not. I don't know how they passed the linting to get to master branch.

Reply:
linter was setup later.